pkg/tracking/service/runs.go

```diff
@@ -62,3 +64,53 @@ func (ts TrackingService) CreateRun(
 
 	return &protos.CreateRun_Response{Run: run}, nil
 }
+
+func (ts TrackingService) UpdateRun(
```

Comment:
This is deviating a bit from the Python store.
Some sensible checks are done here, in the sense that they probably are conceptually valid for both sql and file store. However, this is not what happens in Python.
@jgiannuzzi I'm not sure how much this really matters, please react if you think it does.

Reply:
Well, I'm just wondering for the C interopt side of things.

Reply:
Ok, I was confused but what Dmitry did was right all along.
For interop story, stuff should be in service.